gradle.properties

```diff
@@ -0,0 +1,31 @@
+org.gradle.caching=true
+puppycrawlCheckstyleVersion=8.18
+group=org.ballerinalang
+version=0.1.0-SNAPSHOT
+ballerinaLangVersion=2.0.0-alpha6-SNAPSHOT
```

Comment:
Shall we update to the latest lang and stdlib versions

Reply:
Fixed with [`07c225d`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/07c225de8250609994281d9c953af64e68d31460)